core/src/data/rasterSource.cpp

```diff
@@ -37,37 +37,37 @@ std::shared_ptr<TileData> RasterSource::parse(const TileTask& _task, const MapPr
 
 }
 
+std::shared_ptr<TileTask> RasterSource::createTask(TileID _tileId) {
```

Comment:
`RasterSource` does not need to have its own implementation for `createTask`. It is same as the uber `DataSource::createTask`.